javascript.html.markdown

```diff
@@ -613,6 +613,12 @@ const isEven = (number) => {
 
 isEven(7); // false
 
+// An even shorter way to write a function similar to the above one is:
+// Parenthesis are not required if only one argument is specified;
```

Comment:
The plural of "parenthesis" is "Parentheses": https://data.grammarbook.com/blog/commas/all-about-parentheses/